relay-server/src/statsd.rs

```diff
@@ -14,7 +14,7 @@ pub enum RelayGauges {
     /// w.r.t. to its provisioned capacity.
     AsyncPoolUtilization,
     /// The state of Relay with respect to the upstream connection.
-    /// Possible values are `0` for normal operations and `1` for a network outage.
+    /// Possible values are `0` for normal operations and `100` for a network outage.
```

Comment:
Utilization is not a network outage.

Also missing the tags.

Reply:
I feel blind, sorry for the mistake.